tests/lib.rs

```diff
@@ -229,44 +233,66 @@ fn assert_call_number(test: &str, expected_calls: usize) {
 
 #[test]
 fn test_fdr_control1() {
-    control_fdr("test_fdr_ev_1", "SOMATIC", 0.05, false);
+    control_fdr("test_fdr_ev_1", vec!["SOMATIC"], 0.05, false);
```

Comment:
```suggestion
    control_fdr("test_fdr_ev_1", ["SOMATIC"], 0.05, false);
```
and analogous for the tests below.